core/block_validator_test.go

```diff
@@ -113,8 +113,12 @@ func testHeaderVerificationForMerging(t *testing.T, isClique bool) {
 		}
 		copy(gspec.ExtraData[32:], addr[:])
 
+		// chain_maker has no blockchain to retrieve the TTD from, setting to nil
```

Comment:
I think you can delete all the pre-merge tests

Reply:
We're still supposed to be able to verify backwards across a merge, right? In that case, I think we should leave it in